src/vite.rs

```diff
@@ -139,31 +143,35 @@ window.__vite_plugin_react_preamble_installed__ = true
     }
 }
 
-pub struct Production {
+pub struct Production<'a> {
```

Comment:
this is the struct's lifetime. In order to remove 'static lifetimes, we need to explicitly declare the `str` references lifetimes.
By explicitly declaring the lifetime, it's possible to declare it's fields like 
```rust
pub struct Production<'a> {
    main: ManifestEntry,
    css: Option<String>,
    title: &'a str, // str of lifetime 'a instead of 'static
    lang: &'a str,
    version: String,
}
```

It tells the compiler that this `str` must be valid for the lifetime of the Production instance, instead of the whole application lifetime ('static).